src/structures/idt.rs

```diff
@@ -8,6 +8,18 @@
 // except according to those terms.
 
 //! Provides types for the Interrupt Descriptor Table and its entries.
+//!
+//! # For the users who use this module with the stable Rust
+//!
+//! The following types are not used and not constructable.
```

Comment:
No space or new sentance here. We should also say something like "the following types are opaque and non-constructable instead of function pointers".